spec/scrt.wsdl

```diff
@@ -1,629 +1,629 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I don't know what Atom did to these files to make a simple rename show as a complete rewrite...